core/src/main/java/io/grpc/Attributes.java

```diff
@@ -85,6 +96,17 @@ public Key(String name) {
     public String toString() {
       return name;
     }
+
+    /**
+     * Convenience method for creating instances of {@link Key} class.
```

Comment:
Let's call it a factory method. It's not really just a convenience since the constructor is now private :)